swap/swap.go

```diff
@@ -537,7 +541,17 @@ func (s *Swap) getContractOwner(ctx context.Context, address common.Address) (co
 	return contr.Issuer(nil)
 }
 
-// Deploy deploys the Swap contract
+// BindToContractAt binds an instance of an already existing chequebook contract at address and sets chequebookAddr
+func (s *Swap) BindToContractAt(address common.Address, backend swap.Backend) (err error) {
+	s.contract, err = contract.InstanceAt(address, backend)
+	if err != nil {
+		return err
+	}
+	s.setChequebookAddr(address)
+	return nil
+}
+
+// Deploy deploys the Swap contract and sets the contract address
 func (s *Swap) Deploy(ctx context.Context, backend swap.Backend, path string) error {
```

Comment:
I'm confused as to what the path parameter does.
Is this to satisfy a Interface?, or is it something that is not being used.

And if this is the case what would be the use of such path.

Reply:
I am confused about this as well. Actually removed it before when I was doing refactoring, but when I reverted the refactoring, I also left this one in now. 
Tagging @holisticode , maybe he knows. I would personally remove it, as it does not seem to have a purpose

Reply:
if the tests pass after removing this parameter, i say let's go ahead with this change